src/Peachpie.Runtime/Operators.cs

```diff
@@ -614,6 +614,66 @@ public static string GetItemValue(string value, PhpValue index, bool quiet)
             }
         }
 
+        /// <summary>
+        /// Shortcut for calling <c>ord($s[$i])</c> on a <see cref="PhpValue"/>
+        /// without any extra allocation.
+        /// </summary>
+        public static long GetItemOrdValue(PhpValue value, long index)
+        {
+            switch (value.TypeCode)
+            {
+                case PhpTypeCode.String:
+                    return GetItemOrdValue(value.String, index);
+                case PhpTypeCode.MutableString:
+                    return GetItemOrdValue(value.MutableString, index);
+                default:
+                    var item = value.GetArrayItem(index);
+                    var itemType = item.TypeCode;
```

Comment:
use `IsPhpString()` instead

Reply:
There's no such method, should I implement it? Similarly to `IsPhpArray`?

Reply:
added in https://github.com/peachpiecompiler/peachpie/commit/bb0116bdfa124b772ea1c2779e3cc8cd91f431dc

Reply:
Thanks, updated in f1b7f783f21a6a8e9439d268e1da33251935a066.